cli/project/dockerfiles.go

```diff
@@ -21,7 +21,8 @@ func init() {
 }
 
 type opensourseCtx struct {
-	MajorMinorVersion string
+	Type    string // release or pre-release
```

Comment:
As for me, the style that puts the attribute description on a separate line like [here](https://github.com/tarantool/tvisor/blob/30ba24f8d095b64dad3665ecdc7bf47e91db6d67/supervisor/main.go#L21) is more readable in one hand and the same style is enforced in tarantool core on the other hand. So, if you don't object - do the same. Up to you, I do not insist.

As for me, the style that puts the attribute description on a separate line like [here] (https://github.com/tarantool/tvisor/blob/30ba24f8d095b64dad3665ecdc7bf47e91db6d67/supervisor/main.go#L21) is more readable in one hand ... and the same style is enforced on the other side of the tarantool core. So if you don't mind, do the same. At your discretion, I do not insist.

Reply:
Okay, every comment is reworked to a complete sentence on a separate line.